flink/cr-examples/flinkdeployment/quick-start.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: flink.apache.org/v1beta1
+kind: FlinkDeployment
+metadata:
+  name: session-cluster-quick-start
+spec:
+  flinkConfiguration:
+    license.use: EventAutomationProduction
```

Comment:
The Quick Start sample is supposed to use the "Non Production" license and not the Production license.
Here it should be EventAutomationNonProduction instead of EventAutomationProduction